pkg/tasks/tasks.go

```diff
@@ -0,0 +1,90 @@
+package tasks
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"strings"
+
+	"gopkg.in/yaml.v2"
+)
+
+var (
+	ErrNoTaskFile = errors.New(".draft/tasks.yaml not found")
+)
+
+const (
+	Dependencies = "Dependencies"
+	Setup        = "Setup"
+	Cleanup      = "Cleanup"
+)
+
+type Tasks struct {
+	Dependencies []string
+	Setup        []string
+	Cleanup      []string
+}
+
+// Load takes a path to file where tasks are defined and loads them in tasks
+func Load(path string) (*Tasks, error) {
+	if _, err := os.Stat(path); err != nil {
+		if os.IsNotExist(err) {
+			return nil, ErrNoTaskFile
+		}
+		return nil, err
+	}
+
+	t := Tasks{}
+	tFile, err := ioutil.ReadFile(path)
+	if err != nil {
+		return nil, fmt.Errorf("Error reading task file: %v:%v", path, err)
+	}
+
+	err = yaml.Unmarshal(tFile, &t)
```

Comment:
I know it's just a PoC, but do we want to make this a TOML file instead so it matches draft.toml?